build.sbt

```diff
@@ -3606,6 +3617,93 @@ lazy val `std-microsoft` = project
   .dependsOn(`std-table` % "provided")
   .dependsOn(`std-database` % "provided")
 
+lazy val `std-tableau` = project
+  .in(file("std-bits") / "tableau")
+  .settings(
+    frgaalJavaCompilerSetting,
+    autoScalaLibrary := false,
+    unmanagedExternalZip := {
+      val platform = if (Platform.isWindows) {
+        "windows"
+      } else if (Platform.isMacOS) {
+        "macos"
+      } else if (Platform.isLinux) {
+        "linux"
+      }
+      val arch = if (Platform.isArm64) {
+        "arm64"
+      } else {
+        "x86_64"
+      }
+      new URI(
+        s"https://downloads.tableau.com/tssoftware//tableauhyperapi-java-$platform-$arch-release-main.$tableauVersion.zip"
+      ).toURL()
+    },
+    fetchZipToUnmanaged := {
+      val unmanagedDirectory = (Compile / unmanagedBase).value
+      if (IO.listFiles(unmanagedDirectory).size < 2) { // Heuristic, should have at least hyperapi jar and os-specific one.
+        System.out.println(
```

Comment:
Should we use `ManagedLogger`?

Reply:
In general I didn't see the need for any logging. Somehow CI doesn't like my solution and we can't reproduce locally

Reply:
As you prefer. I think it's useful to see a log for this (at least on debug level) to be able to tell what's happening. I'd especially appreciate a message right before we start a download of the 200Mb file - for fiber connections that's moments, but when you are developing on a train, getting a message explaining to you why the build process just started to block for 5 minutes is actually precious information.

I think it's better to do this via `ManagedLogger` though, as it allows users to manage log levels and gathers logs in one place - print is going to be completely separate from it.